gae_serve/modelserve.yaml

```diff
@@ -0,0 +1,75 @@
+# Copyright 2017 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+---
+swagger: "2.0"
+info:
+  title: "modelserve"
+  description: "Get prediction from sklearn model."
+  version: "1.0.0"
+host: "modelserve.endpoints.PROJECT_ID.cloud.goog"
```

Comment:
Change to `PROJECT_ID.appspot.com` here as well.

Reply:
done.